<launch>
    <group ns="hardware">
        <node name="light_sensors_node" pkg="hardware" type="light_sensors_node" output="screen" />
        <node name="laser_scan_node" pkg="hardware" type="laser_scan_node" output="screen" />
    </group>
</launch>